google-cloud-logging/src/main/java/com/google/cloud/logging/LogDestinationName.java

```diff
@@ -109,6 +109,16 @@ public LogName toLogName(String logId) {
     return null;
   }
 
+  /** Returns ID value associated with {@code LogDestinationName} object */
+  public String getId() {
+    return getValue().toString();
```

Comment:
nit: calling this an "id" is a bit confusing, since you also have `toLogName(logId)`

Maybe this should be called `getDestinationId()`?

Reply:
Yep, makes sense